tests/system/runtimes/test_application.py

```diff
@@ -187,7 +187,7 @@ def _create_vizro_application(
         function = self.project.set_function(
             name=name,
             kind="application",
-            requirements=["vizro", "gunicorn", "Werkzeug==2.2.2"],
+            requirements=["vizro<0.1.32", "gunicorn", "Werkzeug==2.2.2"],
```

Comment:
Add a TODO to remove it once we upgrade pydatic

Reply:
I'm not sure if it's related to our pydantic version or the actual usage of the vizro application.
I'm also not sure if we should remove it because that's not the point of the test